packer/builder/azure/driver_restapi/config.go

```diff
@@ -0,0 +1,141 @@
+package driver_restapi
+
+import (
+	"fmt"
+	"github.com/MSOpenTech/packer-azure/packer/builder/azure/driver_restapi/targets"
+	"github.com/MSOpenTech/packer-azure/packer/builder/azure/utils"
+	"github.com/mitchellh/mapstructure"
+	"github.com/mitchellh/packer/common"
+	"github.com/mitchellh/packer/helper/config"
+	"github.com/mitchellh/packer/packer"
+	"log"
+	"os"
+	"regexp"
+)
+
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+
+	SubscriptionName        string `mapstructure:"subscription_name"`
+	PublishSettingsPath     string `mapstructure:"publish_settings_path"`
+	StorageAccount          string `mapstructure:"storage_account"`
+	StorageAccountContainer string `mapstructure:"storage_account_container"`
```

Comment:
Probably should be called StorageContainer, but ignore if it was an existing setting.